rust/stackable-cockpit/src/platform/stack/spec.rs

```diff
@@ -134,6 +135,18 @@ impl StackSpecV2 {
             });
         }
 
+        if let Some(resource_requests) = &self.resource_requests {
+            resource_requests
+                .warn_when_cluster_too_small("Stack")
+                .await
+                .context(ParseStackResourceRequirementsSnafu)?;
+        };
+
+        if skip_release_install {
+            info!("Skipping release installation during stack installation process");
+            return Ok(());
+        }
```

Comment:
I would say even when you skip installing the release it is important to check

1.) If the stack can run in the ns you are deploying it to
2.) If the cluster is big enough

If the stack does not support the namespace (or is too big) it will definitively not work, regardless if the release is installed by stackablectl or not

Reply:
As discussed: f293deeb96b9787431e82307ae9eda216d919eaa